### PR TITLE
Add multi-arch support stolostron/backlog#26923

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/26923

### CODE REVIEW

**Review by ldpliu — APPROVED** [review]

File: multicluster_engine/cluster_lifecycle/release_images.adoc [hunk 1]
```diff
@@ -64,7 +64,22 @@ You can create a cluster on an architecture that is different from the architect
 
 For example, you might need to create an `x86_64` cluster from a hub cluster that is running on the `ppc64le`, `aarch64`, or `s390x` architecture. If you create the release image with both sets of files, the cluster creation succeeds because the new release image enables the {ocp-short} release registry to provide a multi-architecture image manifest. 
 
-To create the release image, complete steps similar to the following example for your architecture type:
+OpenShift 4.11 and later supports multiple architectures by default. You can use the following `clusterImageSet` to provision a cluster:
```
Comment (dockerymick) [inline multicluster_engine/cluster_lifecycle/release_images.adoc]: I recommend that you use the conref for OpenShift

**Review by dockerymick — APPROVED** [review]
Left a comment, looks good otherwise

File: multicluster_engine/cluster_lifecycle/release_images.adoc (hunk 1 above)
Comment (oafischer) [reply multicluster_engine/cluster_lifecycle/release_images.adoc]: I don't think I see one, which one are you referring to?

Comment (dockerymick) [reply multicluster_engine/cluster_lifecycle/release_images.adoc]: {ocp-short}, which is OpenShift Container Platform

Comment (oafischer) [reply multicluster_engine/cluster_lifecycle/release_images.adoc]: Thanks, fixed!

File: multicluster_engine/cluster_lifecycle/release_images.adoc [hunk 2]
```diff
@@ -64,7 +64,22 @@ You can create a cluster on an architecture that is different from the architect
...
+metadata:
+  labels:
+    channel: fast
+    visible: 'true'
+  name: img4.12.0-multi-appsub
+spec:
+  releaseImage: quay.io/openshift-release-dev/ocp-release:4.12.0-multi
+----
+
+To create the release image for OpenShift images that do not support multiple architectures, complete steps similar to the following example for your architecture type:
```
Comment (dockerymick) [inline multicluster_engine/cluster_lifecycle/release_images.adoc]: OpenShift still used in this line

**Pull request merged**